CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,32 @@
+# v1.9.0 - Changelog since v1.8.2
+
+## Changes by Kind
+
+### Feature
+
+- Adding auto-stamped details like PV, PVC and namespace name from PV description as labels for the PVs/PDs ([#1090](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1090), [@Sneha-at](https://github.com/Sneha-at))
+- Support setting snapshot labels to Images ([#1066](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1066), [@jenting](https://github.com/jenting))
+- Pass in ProvisionedIOPSOnCreate as a parameter for CreateVolume ([#1079](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1079), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Add provisionedThroughput for hyperdisk ([#1101](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1101), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Change iops params directly convert string to int64 ([#1128](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1128), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
#1128 has release note as "NONE", would we want to include it in the CHANGELOG? the same for #1101

Reply:
Do we usually add features in the release note? How about bug fixes?

Reply:
I think if the `release-note` section in PR is "NONE" that means users don't need to be notified of such change and that PR doesn't need to be included in `CHANGELOG`

Reply:
can you update the release note for those two PRs so they match what's shown here

Reply:
I was referring to some previous release, seems like the release note is "NONE" mostly:

* https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1096
* https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1116

Reply:
ahh, sorry for the confusion. for these two PRs:
- #1128
- #1101

I was asking if you want to update their release note so they reflect the feature change we want to communicate to OSS users or, if there's no need to communicate those PRs in release note, remove them from `CHANGELOG-1.9`

Reply:
I updated the release note to:

```
Add provisionedThroughput and provisionedIops for hyperdisk
```

Reply:
Okay I only added the release note to #1101: Add provisionedThroughput for hyperdisk

For https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1128 it is an adjustment of a feature we did not release yet and that feature pr has 'Add provisionedIops for hyperdisk' as release note.